### ISSUE TITLE
Data stale

### ISSUE THREAD

**Issue by radiumsoup** [xcpdex/xcpdex#1]
Trading data is not updating at all. Is this repository the working code or is it just some sort of place where the code gets uploaded to every once in a while after changes have been made and tested on the live site?

**Comment by droplister** [comment]
Yes, something has gone awry with my scheduled updates. Looking into it, but it's up-to-date now.
